src/routes/controllers/articles.controller.js

```diff
@@ -70,33 +70,67 @@ export async function createArticle(req, res) {
  * @returns {Object} res with article object if it exists
  * @returns {Object} res with 404 response if the array is empty
  */
-export async function getArticle(req, res) {
+export async function getSingleArticle(req, res) {
   const { articleId } = req.params;
 
   try {
     const article = await Article.findByPk(articleId, {
       attributes: {
         exclude: ['id', 'userId', 'subscriptionType', 'readTime'],
       },
+    });
+
+    if (article) {
+      return successResponse(
+        res,
+        200,
+        'Article successfully retrieved',
+        article,
+      );
+    }
+    return errorResponse(res, 404, 'Article does not exist');
+  } catch (error) {
+    return errorResponse(res, 500, 'database error', error);
+  }
+}
+
+/**
+ * Fetch all articles
+ * @param {Object} req Express Request Object
+ * @param {Object} res Express Response Object
+ * @returns {Object} res with articles array if it exists
+ * @returns {Object} res with 404 response if the array is empty
+ */
+export async function getAllArticles(req, res) {
+  const currentPage = req.query.page || 1;
+  const limit = req.query.limit || 10;
+  const offset = limit * (currentPage - 1);
+  try {
+    const articles = await Article.findAndCountAll({
+      attributes: {
```

Comment:
we can remove the `attributes` query field.

Reply:
I have removed it